config/locales/hyrax.en.yml

```diff
@@ -727,6 +730,7 @@ en:
         keyword:                   "Keyword"
         lease_expiration_date:     'until'
         related_url:               "Related URL"
+        title:                     "Title"
```

Comment:
"Title" is the default value for title. I added it because I would assume that anywhere using the default would want it to be translated, and this allows for that if it IS used elsewhere. I'm not changing the text that would be displayed for English, which is why I put it into default. It only made sense to me to put it into admin_set if we wanted it to not use the default value of "Title".